src/screens/Customers/AddCustomer.tsx

```diff
@@ -78,26 +114,81 @@ function AddCustomer(props: AddCustomerProps) {
     setCustomerName(event.target.value as string);
   }
 
-  const handleMeterInput = (event: React.ChangeEvent<{ value: unknown }>) => {
+  const handleMeterNumberInput = (event: React.ChangeEvent<{ value: unknown }>) => {
     setMeterNumber(event.target.value as string);
   }
 
+  const handleCustomerNumberInput = (event: React.ChangeEvent<{ value: unknown }>) => {
+    setCustomerNumber(event.target.value as string);
+  }
+
   return (
     <BaseScreen title="Add New Customer" leftIcon="backNav">
-      <form noValidate className={classes.content} onSubmit={() => false}>
-        <TextField label={'Name'} id={'name'} onChange={handleNameInput} />
-        <FormControl variant="outlined" className={classes.formControl}>
-          <InputLabel id="select-tariff-plan-label">Select Tariff Plan</InputLabel>
-          <Select label={"Select Tariff Plan"} id={'select-tariff-plan'} labelId="select-tariff-plan-label" onChange={handleSelectTariffPlan}>
-            {tariffPlans.map((plan) =>
-              <MenuItem key={plan.id} value={plan.id}>{plan.name}</MenuItem>
-            )}
-          </Select>
-        </FormControl>
-        <Checkbox label={'Meter:'} textField={hasMeter} checkboxOnChange={() => setHasMeter(!hasMeter)} textFieldOnChange={handleMeterInput} />
-        <Checkbox label={'Customer is inactive'} checkboxOnChange={() => setCustomerInactive(!customerInactive)} />
-        <Button label={'Add'} onClick={handleSubmit} />
-      </form>
+      <BaseScrollView>
+        <form noValidate onSubmit={() => false}>
+          <div className={classes.textContainer}>
+            <div className={classes.inline}>
+              <div className={classes.nameInput}>
+                <TextField
+                  label={'Name'}
+                  id={'name'}
+                  placeholder={'e.g. Tom'}
+                  onChange={handleNameInput}
+                  required
+                />
+              </div>
+              <div className={classes.numberInput}>
+                <TextField
+                  label={'Number'}
+                  id={'number'}
+                  placeholder={'e.g. 12'}
+                  type="number"
+                  onChange={handleCustomerNumberInput}
+                  required
+                />
+              </div>
+            </div>
+          </div>
+          <div className={classes.selectContainer}>
+            <FormControl required variant="outlined" className={classes.formControl}>
+              <InputLabel>Meter Type</InputLabel>
+              <Select onChange={handleSelectMeterType}>
+                <MenuItem value={MeterType.ANALOG_METER}>Analog Meter</MenuItem>
+                <MenuItem value={MeterType.SMART_METER}>Smart Meter</MenuItem>
+                <MenuItem value={MeterType.NO_METER}>No Meter</MenuItem>
+              </Select>
```

Comment:
Adding the label to match what's in InputLabel helped fix the border issue I think you mentioned

source: https://github.com/mui-org/material-ui/issues/14203#issuecomment-764146829
![image](https://user-images.githubusercontent.com/35501399/114929756-10d0eb00-9de9-11eb-8325-65f88e30f115.png)

Before:
![image](https://user-images.githubusercontent.com/35501399/114929882-365df480-9de9-11eb-912b-1175ae480b2e.png)

After:
![image](https://user-images.githubusercontent.com/35501399/114929822-27774200-9de9-11eb-98c2-dd2b38fa74fd.png)